setup.ts

```diff
@@ -106,6 +106,49 @@ async function mongoDB(): Promise<void> {
   }
 }
 
+//image check for size and format
+
+async function imageValidation(): Promise<void> {
+  const { maxImageSize, maxImageWidth, maxImageHeight, supportedImageFormats } =
+    await inquirer.prompt([
+      {
+        type: "input",
+        name: "maxImageSize",
+        message: "Enter the maximum allowed image size in bytes:",
+        default: process.env.MAX_IMAGE_SIZE || "5242880",
```

Comment:
Remove. This is being governed by the API. The Admin app should have logic to test whether the file is too big based on the API response.

- https://github.com/PalisadoesFoundation/talawa-api/pull/1710